update-dependencies.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# updates Pipfile.lock and regenerates the requirements.txt file
+set -e
+
+# create/update existing venv
+rm -rf venv/
+
+# whatever your preferred version of python is, eLife needs to support python3.6 (Ubuntu 18.04)
+python3.6 -m venv venv
+
+# prefer using wheels to compilation
+source venv/bin/activate
+pip install wheel 
```

Comment:
I could probably get rid of this, only useful on projects where binaries are needed

Reply:
I remember using that to create alternate packages for pypi, but I'm not sure if it's need here, or needed anymore.

Reply:
it's needed in some projects actually. Without it, some things like `cryptography` try to compile their own on the system and usually fail unless a bunch of dev dependencies are there (like `python-dev`, `libpq` etc). I thought simply having it installed on the system was enough but it needs to be there in the virtualenv itself. One of those lessons that was more frustrating than it should have been.